### DISCUSSION TITLE
Welcome to r3dmol Discussions!

### DISCUSSION THREAD

**Discussion by swsoyee** [swsoyee/r3dmol#18] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by dschust-r** [comment]
Not necessarily a topic for discussion but a compliment from us ([protti](https://github.com/jpquast/protti/) developers):
We really like your R package and we were super excited when we found it. Your package is perfect for our types of analyses and was really helpful for writing our [data analysis/structure mapping vignette](https://jpquast.github.io/protti/articles/protein_structure_workflow.html). 

Just wanted to let you know that you did a great job in creating a really useful R package for structural biologists/systems biologists/people in the field of structural mass spectrometry 🥳🔬

**Reply by swsoyee** [reply]
Wow, thank you so much!
I have not been doing research for some years, so I am not at all a user of this package that I developed myself.
However, if there are any problems with this package or any needs, please feel free to create issues, I will do my best to meet the needs of users.

**Reply by dschust-r** [reply]
Thank you! I will let you know in case anything comes up. So far - for our purposes - it is great as is 😃

**Reply by BradyAJohnston** [reply]
Thanks so much for the kind feedback! We really appreciate hearing from people putting it to good use. Please let us know if you have feature requests or if you come across any problem points. :)

**Reply by dschust-r** [reply]
Will do! Thank you you also for all your help so far 😀

**Comment by edornshuld** [comment]
Is there a way to map the m_add_property_labels() function to a button? For example, I'd like a button to turn on and off the atom labels.

**Reply by swsoyee** [reply]
Hi @edornshuld, you could refer to this article:
https://swsoyee.github.io/r3dmol/articles/using-buttons.html
